cmd/plugins/memtierd/Dockerfile

```
@@ -0,0 +1,32 @@
+ARG GO_VERSION=1.21
```

nit: other dockerfiles have 1.20 so I suggest to be aligned with that
```suggestion
ARG GO_VERSION=1.20
```

---

Fixed.